src/_data/docs.yml

```diff
@@ -1,121 +1,237 @@
-sidebar:
+top:
   - title: Getting Started
-    default: true
-    items:
-    - title: Install
-      href: install.html
-    - title: Upgrade
-      href: upgrade.html
-    - title: "Writing a Tiltfile"
-      href: tiltfile_authoring.html
-    - title: "Tilt's Control Loop"
-      href: controlloop.html
-  - title: Tutorial
-    inSectionNav: true
+    href: /index.html
+    hrefMobile: /docs_nav_gettingstarted.html
+    active: gettingstarted
+  - title: Guides
+    active: guides
+    href: /tiltfile_authoring.html
+    hrefMobile: /docs_nav_guides.html
+  - title: Tiltfile & CLI
+    href: /api.html
+    active: reference
+    hrefMobile: /docs_nav_reference.html
+  - title: Tilt API
+    href: https://api.tilt.dev
+    hrefMobile: https://api.tilt.dev
+    flair: new
+
+
+sidebar:
+  gettingstarted:
+  - title: Why Use Tilt?
     items:
     - title: Overview
-      href: tutorial/index.html
-    - title: 1. Preparation (optional)
-      href: tutorial/1-prerequisites.html
-    - title: 2. Launching & Managing Resources
-      href: tutorial/2-tilt-up.html
-    - title: 3. Tilt UI
-      href: tutorial/3-tilt-ui.html
-    - title: 4. Code. Update. Repeat.
-      href: tutorial/4-code-update-repeat.html
-    - title: 5. Smart Rebuilds with Live Update
-      href: tutorial/5-live-update.html
-  - title: Example Projects
-    include: examples
-  - title: Onboarding Your Team
-    items:
+      href: index.html
```

Comment:
Weird Jekyll/Liquid thing, but right now the active page highlight doesn't work for this, if you make it `href: ''` it will. (It ends up seeing the default route as being an empty string rather than `index.html` which breaks the comparison)

Reply:
Ooh, great suggestion. Done!